### DISCUSSION TITLE
How to group words of a single line

### DISCUSSION THREAD

**Discussion by joel1895** [mindee/doctr#1235] [Q&A]
Hi, I wanted to ask how to group words of a single line. I am unable to group individual words.

**Comment by felixT2K** [comment]
Hi @joel1895 :wave:,

Please tack a look at: https://mindee.github.io/doctr/using_doctr/using_models.html#what-should-i-do-with-the-output
You can simply iterate trough the exported results and do whatever you want with it :)
For example filtering for specific words or lines.
